[FIX] range: adapt string XC on sheet rename

LucasLefevre opened the pull request:
## Description:

description of this task, what is implemented and why it is implemented that way.

Task: [TASK_ID](https://www.odoo.com/odoo/2328/tasks/TASK_ID)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_t("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

LucasLefevre committed:
[REF] range: rename parameter

Task: 0

LucasLefevre committed:
[FIX] range: adapt string XC on sheet rename

Task: 0

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/o-spreadsheet/pull/7691.png)](https://mergebot.odoo.com/odoo/o-spreadsheet/pull/7691)

LucasLefevre committed:
[REF] range: add range to NONE

LucasLefevre committed:
[REF] range: add result changeType

LucasLefevre committed:
[IMP] tests: add test ref on deleted sheet

I accidently broke this behavior in a previous code change
and it wasn't tested.
